src/guacd/daemon.c

```diff
@@ -17,6 +17,8 @@
  * under the License.
  */
 
+#define _GNU_SOURCE 1
```

Comment:
Which function is this required for?

Reply:
Required for `pthread_setattr_default_np()`. I'm happy to either add a comment to that effect, or use an alternative function?